content/components/treeview.mdx

```diff
@@ -0,0 +1,595 @@
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Home</Box>                                                                                                                                                                                   | Moves focus to first node without opening or closing a node.                                                                                                                                                                                                                                                                                     |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>End</Box>                                                                                                                                                                                    | Moves focus to the last node that can be focused without expanding any nodes that are closed.                                                                                                                                                                                                                                                    |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>a-z</Box>, <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>A-Z</Box>, all printable characters | <ul><li>Focus moves to the next node with a name that starts with the typed character.</li><li>Search wraps to first node if a matching name is not found among the nodes that follow the focused node.</li><li>Search ignores nodes that are descendants of closed nodes.</li><li>Focus should not move if no nodes match the search.</li></ul> |
+
+#### Typeahead behavior
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+  <Box flex={1}>
+    <p>
+      Focus moves to the next node with a name that starts with the typed character(s). Wait for 300ms from the last key
+      press to stop searching.
+    </p>
+    <p>
+      Typeahead behavior is case insensitive, and any printable character can be used, not just alphanumeric characters.
+      The exception is the “*” character, which is used to expand all nodes.
+    </p>
+    <p>
+      The `Space` key is ignored because it's being reserved for toggling a tree view node's checkbox. Nodes with
+      checkboxes are not supported yet, but they may in the future.
```

Comment:
This conflicts with [proposed documentation in the TreeView API documentation](https://github.com/github/primer/pull/1273/files#diff-6e09f5204050a1c218cd30c0b90bd87a09d2a872eb4b4a56e771e56ae50eb793R19). 

@jscholes: I'd be curious for your opinion here. Should we proactively remove <kbd>Space</kbd> functionality in anticipation of this upcoming functionality?

Reply:
This is no longer a question, resolved in a conversation today.